src/kaleidoscope/device/virtual/Virtual.h

```diff
@@ -86,10 +86,10 @@ class VirtualKeyScanner
 
   bool read_matrix_enabled_;
 
-  KeyState keystates_[matrix_rows * matrix_columns];
-  KeyState keystates_prev_[matrix_rows * matrix_columns];
+  KeyState keystates_[matrix_rows * matrix_columns]; // NOLINT(runtime/arrays)
+  KeyState keystates_prev_[matrix_rows * matrix_columns]; // NOLINT(runtime/arrays)
 
-  bool mask_[matrix_rows * matrix_columns];
+  bool mask_[matrix_rows * matrix_columns]; // NOLINT(runtime/arrays)
```

Comment:
All of these are false positives: both `matrix_rows` and `matrix_columns` are template arguments. We happen to force them into a constexpr first for readability, which cpplint can't follow.